docs/reference/commandline/node_rm.md

```diff
@@ -30,6 +31,20 @@ Example output:
     $ docker node rm swarm-node-02
     Node swarm-node-02 removed from swarm
 
+This command will only remove nodes that are in the down state. Attempting to remove
+an active node will result in an error:
+
+    $ docker node rm swarm-node-03
+    Error response from daemon: rpc error: code = 9 desc = node swarm-node-03 is not down and can't be removed
+
+If a node is compromised, starts misbehaving, or access to it is lost such that a
+clean shutdown of the node is not possible, you can use the force option.
```

Comment:
Correct. Added text to say that managers have to be demoted before they can be removed.